Session/Open Groups/JoinOpenGroupVC.swift

```diff
@@ -166,8 +166,24 @@ final class JoinOpenGroupVC: BaseVC, UIPageViewControllerDataSource, UIPageViewC
                     Storage.shared.writeAsync { db in
                         try MessageSender.syncConfiguration(db, forceSyncNow: true).retainUntilComplete() // FIXME: It's probably cleaner to do this inside addOpenGroup(...)
                     }
+                    
+                    let maybeThread: SessionThread? = Storage.shared.write { db in
+                        let threadId: String = OpenGroup.idFor(roomToken: roomToken, server: server)
+                        return try SessionThread.fetchOrCreate(db, id: threadId, variant: .openGroup)
+                    }
 
                     self?.presentingViewController?.dismiss(animated: true, completion: nil)
+                    
+                    if let thread = maybeThread, shouldOpenCommunity {
+                        SessionApp.presentConversation(
+                            for: thread.id,
+                            threadVariant: thread.variant,
```

Comment:
Should be able to provide `OpenGroup.idFor(roomToken: roomToken, server: server)` and `threadVariant: .openGroup` directly here instead of pulling them from the thread